adapter/internal/discovery/xds/enforcercallbacks/enforcer_callbacks.go

```diff
@@ -98,3 +108,12 @@ func (cb *Callbacks) OnStreamDeltaResponse(id int64, req *discovery.DeltaDiscove
 func (cb *Callbacks) OnStreamDeltaRequest(id int64, req *discovery.DeltaDiscoveryRequest) error {
 	return nil
 }
+
+func getNodeIdentifier(request *discovery.DiscoveryRequest) string {
+	metadataMap := request.Node.Metadata.AsMap()
```

Comment:
If someone overrode the metadata flag in the envoy startup cmd,  `request.Node.Metadata` could be null, isn't it? Shouldn't we handle it

Reply:
It does not become null even though the Metadata property is not assigned. I have tested it too.